docs/charter.md

```diff
@@ -46,25 +51,23 @@ The _CPANSec_ intends to be a forum for coordinating and assisting in resolving
 - Share and document best security practices to authors and users;
 - Raise awareness to security threats and mitigations in Perl applications that rely on the CPAN ecosystem;
 - Act as a trusted intermediator for reporting potential security vulnerabilities to distributions, including managing and responsible disclosure of embargoed security information for a reasonable amount of time pertaining non-core distributions on CPAN, and communication with package managers and distribution channels;
-  - ~~In what ways / For how long? (e.g. max 14 days?) /  explain!~~
-    - Rationale: Let's keep specific embargo durations out of the charter, so we don't have to re-publish the charter when this changes. (garu)
 - Provide a forum for discussing and coordinating around supply chain security, integrity, metadata and the communication of these, including but not limited to:
-  - SBOM and other ways for tracking and communicating software changes, including interaction with external tooling and sources of security advisories;
+  - SBOM and other ways for tracking and communicating software and metadata changes, including interaction with external tooling and sources of security advisories;
   - The Update Framework (TUF) integration;
   - Third-party vulnerability scanning support;
 
 
-
 ### In-scope
 
 The _CPANSec_ concerns itself with a limited domain:
 
-- Distributions published on CPAN, including dual-life core modules;
-- Supply chain security, including CPAN/MetaCPAN and PAUSE
-  - [Artifact Transparency Logs](https://transparency.dev) like sigstore or sigsum
-  - Tooling and infrastructure for establishing chain-of-trust
```

Comment:
"chain-of-trust" is an important word that should probably be kept at least in some item of the list, shouldn't it? /cc @stigtsp

Reply:
I moved it to the projects section, mostly because it fall under the Provenance and Supply Chain Security project, where it is mentioned.

I'm fine with adding it back to this list too, though! Happy to do so, even.